controllers/gatekeeper_controller_test.go

```diff
@@ -55,18 +58,24 @@ func TestDeployWebhookConfigs(t *testing.T) {
 	// MutatingWebhookConfiguration enabled
 	gatekeeper.Spec.ValidatingWebhook = &webhookEnabled
 	gatekeeper.Spec.MutatingWebhook = &webhookEnabled
-	deleteAssets, applyAssets = getStaticAssets(gatekeeper)
-	g.Expect(applyAssets).To(ContainElement(ValidatingWebhookConfiguration))
+	deleteAssets, applyAssets, webhookAssets = getStaticAssets(gatekeeper)
 	g.Expect(applyAssets).To(ContainElements(mutatingStaticAssets))
+	g.Expect(applyAssets).NotTo(ContainElement(ValidatingWebhookConfiguration))
+	g.Expect(applyAssets).NotTo(ContainElement(MutatingWebhookConfiguration))
+	g.Expect(webhookAssets).To(ContainElement(ValidatingWebhookConfiguration))
+	g.Expect(webhookAssets).To(ContainElement(MutatingWebhookConfiguration))
 	g.Expect(deleteAssets).NotTo(ContainElement(ValidatingWebhookConfiguration))
 	g.Expect(deleteAssets).NotTo(ContainElements(mutatingStaticAssets))
 
 	// ValidatingWebhookConfiguration enabled
 	// MutatingWebhookConfiguration disabled
 	gatekeeper.Spec.ValidatingWebhook = &webhookEnabled
 	gatekeeper.Spec.MutatingWebhook = &webhookDisabled
-	deleteAssets, applyAssets = getStaticAssets(gatekeeper)
-	g.Expect(applyAssets).To(ContainElement(ValidatingWebhookConfiguration))
+	deleteAssets, applyAssets, webhookAssets = getStaticAssets(gatekeeper)
+	g.Expect(webhookAssets).To(ContainElement(ValidatingWebhookConfiguration))
+	g.Expect(webhookAssets).NotTo(ContainElement(MutatingWebhookConfiguration))
+	g.Expect(applyAssets).NotTo(ContainElement(ValidatingWebhookConfiguration))
+	g.Expect(applyAssets).NotTo(ContainElement(MutatingWebhookConfiguration))
 	g.Expect(applyAssets).NotTo(ContainElements(mutatingStaticAssets))
 	g.Expect(deleteAssets).NotTo(ContainElement(ValidatingWebhookConfiguration))
 	g.Expect(deleteAssets).To(ContainElements(mutatingStaticAssets))
```

Comment:
Need a case for `MutatingWebhookConfiguration`

Reply:
Done